config/wpcalypso.json

```diff
@@ -91,6 +91,7 @@
 		"my-sites/checkout/web-payment/basic-card": false,
 		"network-connection": true,
 		"p2/p2-plus": true,
+		"p2/settings/preapproved-domains": false,
```

Comment:
Tiny detail but I'd say we don't need to add the `/settings` part. We hopefully won't have that many feature flags related to settings. 😅

Reply:
Could we enable it for everything except production? So we could test on `stage` in the meantime.

Reply:
Done. Renamed feature flag, and turned on for all except `production` and `horizon` (since public).